app/helpers/sessions_helper.rb

```diff
@@ -20,6 +28,11 @@ def store_location
     session[:return_to] = request.url
   end
 
+  def sign_out
+    session[:userinfo] = nil
+    redirect_to "https://thoughtworks.okta.com/login/signout?fromURI=#{ENV['URL']}"
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.